frontend/src/styles/base/layout.css

```diff
@@ -0,0 +1,16 @@
+body {
+  background-color: #f6f2f0 !important;
+  height: 100vh;
+  width: 100%;
+  margin: 0;
```

Comment:
would you mind adding a bit of margin? otherwise the posts are all really close to the border of the screens! thanks